requirements-dev.txt

```diff
@@ -18,4 +18,5 @@ pytest-cov==2.5.1
 six==1.11.0
 wrapt==1.10.11
 requests==2.20.1
-flask==1.0.2
\ No newline at end of file
+flask==1.0.2
+adal==1.2.2
```

Comment:
looks like this `adal==1.2.2` is an external library only used by msatp. you should put this in requirements.txt instead and also in setup.py.

Reply:
I have removed adal library for this file and added to requirements.txt and setup.py